app/internal/page/app.go

```diff
@@ -175,16 +169,30 @@ func App(
 
 	handleRoot(paths.Start, None,
 		Guidance(tmpls.Get("start.gohtml"), paths.Auth, nil))
-
 	handleRoot(paths.Dashboard, RequireSession,
 		Dashboard(tmpls.Get("dashboard.gohtml"), lpaStore))
 
+	handleRoot(paths.CertificateProviderStart, None,
+		CertificateProviderStart(tmpls.Get("certificate_provider_start.gohtml"), lpaStore))
+	handleRoot(paths.CertificateProviderLogin, None,
+		CertificateProviderLogin(logger, oneLoginClient, sessionStore, random.String))
+	handleRoot(paths.CertificateProviderLoginCallback, None,
+		CertificateProviderLoginCallback(tmpls.Get("identity_with_one_login_callback.gohtml"), oneLoginClient, sessionStore, lpaStore))
+	handleRoot(paths.CertificateProviderYourDetails, RequireCertificateProviderSession,
+		Guidance(tmpls.Get("certificate_provider_your_details.gohtml"), "", lpaStore))
```

Comment:
I want to move the certificate provider stuff to it's own package, and rename `page`, but not sure what to call them at the moment other than `certifypage` (I think `certificateproviderpage` would be crazy, but maybe?) and `donorpage`

Reply:
In keeping with the action focus of`donorpage` maybe `certifierpage`? Seeing it written down I don't like it much.   although I gues technically following that logic `donorpage` would actually be `donatorpage` which is also not great 🤔.

`certificateproviderpage` is long but if we want to follow the actor+page pattern it fits 🤷